### DISCUSSION TITLE
Workshop is ready!

### DISCUSSION THREAD

**Discussion by Beelzenef** [Beelzenef/workshop_flutter#1] [Announcements]
This workshop is ready for action

Includes:

- multiple pages
- user page
- orders history
- pay or empty cart

**Comment by KamiKeys** [comment]
Oh! I didn't know this feature existed! How cool! 🚀

**Comment by marcoablanco** [comment]
Is a mac necessary to do it?

**Reply by Beelzenef** [reply]
no need! you can develop with flutter on the following OS:

- Windows
- Mac
- Linux
- ChromeOS

more info on [official docs](https://docs.flutter.dev/get-started/install)!

**Comment by KamiKeys** [comment]
> This workshop is ready for action
> 
> 
> 
> Includes:
> 
> 
> 
> - multiple pages
> 
> - user page
> 
> - orders history
> 
> - pay or empty cart

Will new features be added in the future for the repository?

**Reply by Beelzenef** [reply]
already are!
